bin/yb-ctl

```diff
@@ -1409,11 +1521,10 @@ class ClusterControl:
         print("Setup Redis successful.")
 
     def run_cluster_wide_ysql_initdb(self):
-        print("Initializing cluster. This may take up to a minute.")
+        print("Initializing cluster.")
```

Comment:
Let's remove this line now that initdb will be run from master and this essentially does nothing. In case of existing cluster where we will run initdb, you're already printing "YSQL has been enabled on an existing cluster, running initdb" right before this.